packages/container/lib/index.d.ts

```diff
@@ -0,0 +1,17 @@
+interface Container {
```

Comment:
I stubbed out a more limited interface for this in ember-glimmer we should unify